phoenicis-apps/src/main/java/org/phoenicis/apps/BackgroundRepository.java

```diff
@@ -59,4 +59,19 @@ public void getScript(List<String> path, Consumer<ScriptDTO> callback, Consumer<
     public void setFilter(CombinedAppsFilter filter) {
         delegatedRepository.setFilter(filter);
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
You might want to take a look at this [stackoverflow](https://stackoverflow.com/questions/10912646/hashcodebuilder-and-equalsbuilder-usage-style).